packages/kotti-ui/source/kotti-navbar/components/NavbarQuickLink.vue

```diff
@@ -1,26 +1,11 @@
 <template>
 	<div class="kt-navbar__quick-links">
-		<div
-			v-if="!isNarrow"
-			class="kt-navbar__quick-links__title"
-			v-text="title"
-		/>
-		<a
-			v-for="(item, index) in links"
-			:key="index"
-			class="kt-navbar__quick-links__link"
-			:href="item.link"
-			rel="noopener noreferrer"
-			target="_blank"
-		>
+		<div v-if="!isNarrow" class="kt-navbar__link-title" v-text="title" />
```

Comment:
not consistent with what we did elsewhere

this should be treating `kt-navbar__link` as the element (as if it is its own component

so it becomes `kt-navbar-link__title` and `kt-navbar-link` and `kt-navbar-link__tooltip`